### ISSUE TITLE
Dividing the objects

### ISSUE THREAD

**Issue by Tyderf** [Tyderf/ForbiddenIsland#1]
Do you want to do the bottom 4 and I can do the top 4?

**Comment by joe-crick** [comment]
Yes. Also, @Tyderf, please @ reference me when you create an issue---that way I get notified by email.
